src/main/java/org/broadinstitute/hellbender/tools/funcotator/TranscriptSelectionMode.java

```diff
@@ -246,8 +246,8 @@ public ComparatorByProteinCodingStatus(){}
         @Override
         public int compare( final GencodeFuncotation a, final GencodeFuncotation b ) {
             // Is it protein coding?
-            final boolean isAProteinCoding = a.getGeneTranscriptType() == GencodeGtfFeature.GeneTranscriptType.PROTEIN_CODING;
-            final boolean isBProteinCoding = b.getGeneTranscriptType() == GencodeGtfFeature.GeneTranscriptType.PROTEIN_CODING;
+            final boolean isAProteinCoding = (a.getGeneTranscriptType() != null) && a.getGeneTranscriptType().equals(GencodeGtfFeature.KnownGeneBiotype.PROTEIN_CODING.toString());
+            final boolean isBProteinCoding = (b.getGeneTranscriptType() != null) && b.getGeneTranscriptType().equals(GencodeGtfFeature.KnownGeneBiotype.PROTEIN_CODING.toString());
```

Comment:
Minor point: you wouldn't need the explicit null checks here if you reversed the order of the comparison. Eg.,  `GencodeGtfFeature.KnownGeneBiotype.PROTEIN_CODING.toString().equals(b.getGeneTranscriptType())`

Reply:
Yup.  Fixed.